pandas/tests/scalar/test_timedelta.py

```diff
@@ -128,6 +128,61 @@ def test_unary_ops(self):
         assert abs(-td) == td
         assert abs(-td) == Timedelta('10d')
 
+    def test_mul(self):
+        # GH#19365
+        td = Timedelta(minutes=3)
+
+        result = td * 2
+        assert result == Timedelta(minutes=6)
+
+        result = td * np.int64(1)
+        assert isinstance(result, Timedelta)
+        assert result == td
+
+        result = td * 1.5
+        assert result == Timedelta(minutes=4, seconds=30)
+
+        result = td * np.array([3, 4], dtype='int64')
+        expected = np.array([9, 12], dtype='m8[m]').astype('m8[ns]')
+        tm.assert_numpy_array_equal(result, expected)
+
+        with pytest.raises(TypeError):
+            td * pd.Timestamp(2016, 1, 2)
+
+    def test_add_datetimelike(self):
+        # GH#19365
+        td = Timedelta(10, unit='d')
+
+        result = td + datetime(2016, 1, 1)
+        assert result == pd.Timestamp(2016, 1, 11)
+
```

Comment:
I KNOW we have tests for this, is there a reason you are duplicating? (rather than moving those to here)?

Reply:
test_ops_scalar. I have not problem adding tests, but please don't duplicate.

Reply:
> I KNOW we have tests for this, is there a reason you are duplicating? (rather than moving those to here)?

ATM there is NO test for `Timedelta(10, unit='d') + np.timedelta64(-4, 'D')` (try it; it returns a timedelta64).  More generally, there are no add/sub tests in test_timedelta.